src/app/scheduler/scheduled-process/scheduled-process.component.html

```diff
@@ -30,7 +30,7 @@
               <th></th>
             </tr>
           </thead>
-          <tr *ngFor="let schedule of scheduleData">
+          <tr *ngFor="let schedule of scheduleData" class="{{(!schedule.enabled) ? 'fade' : ''}}">
```

Comment:
We must use angular way to apply css class dynamically. use `NgClass` instead of simple `class` condition